package.json

```diff
@@ -4,6 +4,10 @@
   "description": "CS96 Dev Team first project -- a text-based communication tool, driven by Node.js",
   "main": "server.js",
   "dependencies": {
+    "getconfig": "2.1.0",
+    "node-uuid": "1.2.0",
+    "socket.io": "1.3.7",
```

Comment:
Why the downgrade in socket.io version? Did this happen for any reason? If not, we should 

`npm install --save socket.io@latest`

Reply:
We spent a long time trying to make our code work with the new 2.0.4 version, and failed. There are breaking changes between the 2 versions, and it was easier to rectify the changes in our own code than the webRTC library we are using (though we did try it for a long time). We did not find any features in 2.0.4 that we need right now, so this seemed reasonable. Is there anything specific in v2 you need or is it just a general preference to work on the most up-to-date version?

Reply:
got it. I looked at the diff between the two versions and it looks like it's not super critical to use the latest version, especially if it'll be a time sink in the short term